<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>12305</string>
  </property>
  <widget class="QPushButton" name="add_user_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>add_user</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_profile_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>query_profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modify_profile_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>modify_profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_train_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>add_train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="release_train_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>release_train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_train_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>query_train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_train_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>delete_train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_ticket_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>query_ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_transfer_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>query_transfer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy_ticket_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>buy_ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_order_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>query_order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refund_ticket_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>refund_ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clean_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
